kalite/project/settings/default.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+DEFAULT_USER_SETTINGS_FILE = os.path.join(os.environ["KALITE_HOME"], 'settings.py')
+
+# Check if default user file exists and load settings from there
+if os.path.isfile(DEFAULT_USER_SETTINGS_FILE):
+    execfile(DEFAULT_USER_SETTINGS_FILE)
+
+# If not, write a new custom user settings file and continue loading with
+# defaults.
+else:
+    with open(DEFAULT_USER_SETTINGS_FILE, "w") as my_settings:
+        my_settings.write(
+            "# This is the default location where the kalite command finds its settings\n"
```

Comment:
Really? I think multiline strings make sense for when we need strings that have newlines. Such as this series of strings right here.